config/initializers/carrierwave.rb

```diff
@@ -10,6 +10,7 @@
   config.set(:storage_access_key, '')
   config.set(:storage_secret_key, '')
   config.set(:storage_region, '')
+  config.set(:uploader, UploadUploader)
```

Comment:
Make sure it's a string, as you can pass only strings in envs (or format it)

Reply:
This one will not be configured from env

Reply:
Maybe I'll disable env for this later